# expressed as * 1E7
int32 Latitude
# expressed as * 1E7
int32 Longitude
# expressed as * 1000 (millimeters). The reference fram is set by other nodes
int32 Altitude
# milliseconds since system boot)
uint32 Timestamp
# heading as (rad)
float32 YawAngle
# reference system 6 = baro 11 = sonar
uint16 frame